lib/testLauncher/SuitestLauncher.js

```diff
@@ -61,7 +62,8 @@ class SuitestLauncher {
 							shell: true,
 							env: {
 								...process.env,
-								[envVars.SUITEST_LAUNCHER_PROCESS]: true,
+								[envVars.SUITEST_LAUNCHER_MAIN_PROCESS]: false,
+								[envVars.SUITEST_LAUNCHER_CHILD_PROCESS]: true,
```

Comment:
Maybe lets change SUITEST_LAUNCHER_PROCESS to type string it in scope of this ticket? Give it some constant values "main", "child".

Reply:
there are 2 variables to distinguish Launcher from user script and main process from child (as 
for child process we logging in main process we need to avoid double '    >' prefixes ) @andriichumak what do you think about @givehug proposal (anyway env veriables are string and true false are strings as well)?

Reply:
OK, let's have one variable with enumerated values. But make it **launcher** and **lib**, because our lib can be used without launcher as well. Also, make sure that **lib** is a default value when none provided.